lib/api/WebApi.js

```diff
@@ -137,19 +155,34 @@ async function getUsersByIds(ids) {
     return new Map();
   }
   const options = { data: { id: idsArray } };
-  const users = await apiClient.fetch('/users/byId', options);
+  let users = await apiClient.fetch('/users/byId', options);
+
+  const usersSchema = Joi.array().items(
```

Comment:
ideally we'd just share these schema definitions directly between frontend and backend - but that can wait.